src/ui/components/shared/Login/Login.tsx

```diff
@@ -1,51 +1,139 @@
+  const [error, setError] = useState<string | boolean>(false);
+
+  const onEnterpriseLogin = async () => {
+    const resp = await query({
+      query: GET_CONNECTION,
+      variables: {
+        email,
+      },
+    });
+
+    if (resp.data?.auth.connection) {
+      loginWithRedirect({
+        connection: resp.data.auth.connection,
+        appState: { returnTo: window.location.pathname + window.location.search },
+      });
+    } else {
+      setError(resp.errors?.find(e => e.message)?.message || true);
+    }
+  };
+
+  return (
+    <div className="space-y-6">
+      <p className="text-center text-base">
+        Enter your email to be redirected to your SSO provider.
+      </p>
+      {error ? (
+        <p className="text-center text-base text-red">
+          {typeof error === "string" ? error : "We couldn't find an SSO provider for your email."}
+        </p>
+      ) : null}
+      <div className="flex flex-row space-x-3">
+        <input
+          type="email"
+          className="flex-grow rounded-md"
+          placeholder="user@company.com"
+          value={email}
+          onKeyPress={e => (e.key === "Enter" ? onEnterpriseLogin() : null)}
+          onChange={e => setEmail(e.currentTarget.value)}
+        />
+
+        <PrimaryLgButton color="blue" onClick={onEnterpriseLogin} className="justify-center">
+          Sign in
+        </PrimaryLgButton>
+      </div>
+      <button
+        className="w-full justify-center text-primaryAccent underline text-sm font-bold"
+        onClick={onLogin}
+      >
+        Sign in with Google
+      </button>
+    </div>
+  );
+}
+
+function SocialLogin({
+  onShowSSOLogin,
+  onLogin,
+}: {
+  onShowSSOLogin: () => void;
+  onLogin: () => void;
+}) {
+  return (
+    <div className="space-y-6">
+      {isTeamMemberInvite() ? <h1 className="text-2xl font-extrabold">Almost there!</h1> : null}
+      <div className="text-base space-y-4 self-start">
+        {isTeamMemberInvite() ? (
+          <p>In order to join your team, we first need you to sign in.</p>
+        ) : (
+          <>
+            <p className="text-center">
+              Replay captures everything you need for the perfect bug report, all in one link.{" "}
+              <a href="https://replay.io" className="underline pointer-hand">
+                Learn more
+              </a>
+            </p>
+            <p></p>
+          </>
+        )}
+      </div>
+      <PrimaryLgButton color="blue" onClick={onLogin} className="w-full justify-center">
+        Sign in with Google
+      </PrimaryLgButton>
+      <button
+        className="w-full justify-center text-primaryAccent underline text-sm font-bold"
+        onClick={onShowSSOLogin}
+      >
+        Enterprise Users: Sign in with SSO
+      </button>
+    </div>
+  );
+}
+
 export default function Login() {
   const { loginWithRedirect } = useAuth0();
-  const forceOpenAuth = new URLSearchParams(window.location.search).get("signin");
```

Comment:
I've dropped support for this since it doesn't make sense any longer. It used to start the Auth0 login flow but we've cut that out and the user chooses the IdP here instead with this PR.